envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -3,6 +3,16 @@ stack_env: example_ci-ceph_swift
 
 state_path_base: /opt/stack/data
 
+swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
+swift_undercloud_floating_ip: "{{ swift_floating_ip }}"
+swift_fqdn: "swift.{{ fqdn }}"
+
+etc_hosts:
+  - name: "{{ fqdn }}"
+    ip: "{{ undercloud_floating_ip }}"
+  - name: "{{ swift_fqdn }}"
+    ip: "{{ swift_undercloud_floating_ip }}"
```

Comment:
For /etc/hosts on our Openstack nodes, it can only be written by https://github.com/blueboxgroup/ursula/blob/master/roles/common/tasks/networking.yml which is using a template:
```
- name: hosts file
  template: src=etc/hosts dest=/etc/hosts owner=root group=root mode=0644
            backup=yes
  when: etc_hosts is defined
  tags: etc_hosts
```

```
{% for entry in etc_hosts -%}
{{ entry.ip }} {{ entry.name }}
{% endfor %}
```

Btw, it seems the logging part you mentioned doesn't have any impact to our ursula test, right?